sdk/inc/azure/storage/az_storage_blobs.h

```diff
@@ -33,16 +33,24 @@
  */
 static az_span const AZ_STORAGE_API_VERSION = AZ_SPAN_LITERAL_FROM_STR("2019-02-02");
 
+/**
+ * @brief Azure Storage Blobs blob client options
+ * @remark Allows customization of the blob client.
+ */
 typedef struct
 {
-  az_http_policy_retry_options retry;
+  az_http_policy_retry_options retry_options; /**< Optional values used to override the default retry policy options **/
```

Comment:
I probable would have dropped _options from these field names since they're already inside an "options" struct. But, up to you; there not public.

Reply:
It felt odd in code to have retry representing a "...retry_options" structure.  "retry" didn't imply to me that its a structure of options.  I agree its private and should we decide to make it public we can certainly name it differently then

Reply:
For discussion sake, how about being more verbose, `retry_policy_options`?

Reply:
Isn't this field public? It isn't part of the nested `_internal` struct.